src/uu/date/src/date.rs

```diff
@@ -274,21 +273,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         for date in dates {
             match date {
                 Ok(date) => {
-                    // TODO - Revisit when chrono 0.5 is released. https://github.com/chronotope/chrono/issues/970
-                    let tz = match std::env::var("TZ") {
-                        // TODO Support other time zones...
-                        Ok(s) if s == "UTC0" || s.is_empty() => Tz::Etc__UTC,
-                        _ => match get_timezone() {
-                            Ok(tz_str) => tz_str.parse().unwrap(),
-                            Err(_) => Tz::Etc__UTC,
-                        },
-                    };
-                    let offset = tz.offset_from_utc_date(&Utc::now().date_naive());
-                    let tz_abbreviation = offset.abbreviation();
-                    // GNU `date` uses `%N` for nano seconds, however crate::chrono uses `%f`
-                    let format_string = &format_string
-                        .replace("%N", "%f")
-                        .replace("%Z", tz_abbreviation.unwrap_or("UTC"));
+                    let format_string = custom_time_format(&format_string);
```

Comment:
clippy complains about this line:
```suggestion
                    let format_string = custom_time_format(format_string);
```